src/Me/components/Input/Input.test.js

```diff
@@ -1,10 +1,12 @@
+import { render } from '@testing-library/react';
 import React from 'react';
 import Input from './';
-import { render } from '@testing-library/react';
 
 describe('Input component', () => {
   it('renders properly', () => {
-    const { container } = render(<Input placeholder="test" />);
+    const { container } = render(
+      <Input placeholder="test" value="test" onChange={() => {}} />
```

Comment:
Why we need `value` and `onChange` (and if any the `placeholder` also) in the test? Unless we test them or they're mandatory.

Reply:
adding `value` attr with a fallback of a non nullish value to an `input` element will turn it into a 'controlled input'.
'controlled input' expect `onChange` to be pass as a prop.